src/test/functional/mutationServices.test/tagsMutationService-update.integration.ts

```diff
@@ -41,12 +41,6 @@ describe('updateTag Mutation: ', () => {
   });
 
   beforeEach(async () => {
-    // Mock Date.now() to get a consistent date for inserting data
-    clock = sinon.useFakeTimers({
-      now: updateDate,
-      shouldAdvanceTime: true,
-    });
-
```

Comment:
wait - is the usage of `sinon.useFakeTimers` in the test file above impacting this test file?

Reply:
it shouldn't be... I see `clock.restore` called in the`afterAll` hook

Reply:
Oh, this looks like it was duplicated in `beforeAll` and `beforeEach`, hence removal

Reply:
> wait - is the usage of sinon.useFakeTimers in the test file above impacting this test file?

@jpetto no, two separate files, each calling `sinon.useFakeTimers` more than once